src/ir_parser.c

```diff
@@ -780,6 +783,8 @@ static bool parse_instruction_or_branch(IRParser *p) {
       if (p->tok_type != tk_number) ERR_AT(i_loc, "Expected physical register after REGISTER");
       INSTRUCTION(reg, IR_REGISTER);
       ir_insert(p->context, reg);
+      // TODO: Type of IR_REGISTER?
```

Comment:
An IR_REGISTER instruction should probably be contain a type/size. The IR parser is horribly broken atm anyway, so that’s something we’ll have to fix at a later point in time...